src/ScrumMaster/Channel/ChannelResult.php

```diff
@@ -53,7 +53,7 @@ public function ticketsAssignedToPeople(): array
             }
         }
 
-        return $tickets;
+        return $this->replaceEmptyKeyArray($tickets);
```

Comment:
What about `replaceEmptyKeyArray($tickets, $from = '', $to = 'None');`?

Reply:
Sure!